java/org/contikios/cooja/util/StringUtils.java

```diff
@@ -192,4 +193,15 @@ public static boolean saveToFile(File file, String text) {
       return false;
     }
   }
+
+  public static ArrayList<String> splitOnNewline(String commands) {
+    final ArrayList<String> cmdList = new ArrayList<>();
+    for (String cmd: commands.split("\n")) {
+      if (cmd.trim().isEmpty()) {
```

Comment:
I think this should store the `trim()` result in `cmd` as was previously done in `java/org/contikios/cooja/mote/BaseContikiMoteType.java:261` ?

Reply:
Nice catch! Fixed.